core/main/src/Core/Particle.ts

```diff
@@ -408,6 +410,11 @@ export class Particle implements IParticle {
         this.mover.move(delta);
     }
 
+    public moveXY(x: number, y: number): void {
```

Comment:
Handle this with `mover` property where needed

Reply:
Just thought to do it like the move(delta) above. But using the mover.moveXY for a particle works too.